### PR TITLE
HOTT-3312: Support reporting bucket prefixes

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-3312

### What?

I have added/removed/altered:

- [x] Adds a favicon
- [x] Move to server-side filtering of object keys

### Why?

I am doing this because:

- We were excluding a bunch of keys because we were limited to 1000 and the reporting bucket is used for caching source data when generating the daily file for HMRC indicating changes in the ETL file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,8 @@
 <html>
 
 <head>
-  <title>Reports</title>
+  <link rel="icon" type="image/png" href="https://reporting.trade-tariff.service.gov.uk/favicon.ico">
```
Comment (jebw) [inline index.html:5]: Could this just be `"/favicon.ico"` ?

**Review by jebw — APPROVED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (willfish) [reply index.html:5]: I honestly don't know

Comment (willfish) [reply index.html:5]: Turns out I can't

**Pull request merged**